iib/workers/tasks/build_create_empty_index.py

```diff
@@ -86,16 +87,30 @@ def handle_create_empty_index_request(
 
     _update_index_image_build_state(request_id, prebuild_info)
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
-        if is_image_fbc(from_index_resolved):
-            err_msg = 'File-Based catalog image type is not supported yet.'
-            log.error(err_msg)
-            raise IIBError(err_msg)
         set_request_state(request_id, 'in_progress', 'Checking operators present in index image')
 
         operators = _get_present_operators(from_index_resolved, temp_dir)
```

Comment:
Q: Is there anyhow taken care of how to get operators when it is FBC image?

Reply:
_get_present_operators() function hides the implementation details well, and  takes care of getting operators even index image is FBC.

More information: the function calls is as follows:
_get_present_operators -> grpcurl_get_db_data -> opm_serve_from_index()
opm_serve_from_index() function determines if the index image is fbc or sqlite based and accordingly calls opm_serve or opm_registry_serve() respectively

Reply:
thanks for explaining